pkg/ingress/kube/annotations/annotations.go

```diff
@@ -138,6 +138,10 @@ type AnnotationHandler interface {
 	RouteHandler
 	TrafficPolicyHandler
```

Comment:
It is suggested to extend this implementation, rather than creating a separate file.
https://github.com/alibaba/higress/blob/main/pkg/ingress/kube/annotations/downstreamtls.go

Reply:
ok,get it,i will get it a try ,thanks for your review.

Reply:
Thank you for contributing